### PR TITLE
User can select three difficulties when setting up the game (no error handling).

### PR DESCRIPTION
**Pull request by GeorgeSmith-Sweeper**
Users can choose "Easy", "Medium", and "Hard" when setting up the game.
No error handling has been created.
All tests cover the "happy" path.

### CODE REVIEW

File: src/test/java/com/EighthLight/app/TestConfig.java [hunk 1]
```diff
@@ -168,6 +168,17 @@ void selecting1ForPlayerDifficultySetsTheAiDifficultlyToEasy() {
         String difficulty = config.getDifficulty();
 
         assertEquals("Easy", difficulty);
+
+    }
+
+    @Test
+    void selecting2ForPlayerDifficultySetsTheAiDifficultlyToMedium() {
```
Comment (GeorgeSmith-Sweeper) [inline src/test/java/com/EighthLight/app/TestConfig.java]: "Happy" path test for selecting medium difficulty.

File: src/test/java/com/EighthLight/app/TestConfig.java [hunk 2]
```diff
@@ -168,6 +168,17 @@ void selecting1ForPlayerDifficultySetsTheAiDifficultlyToEasy() {
...
+        aiDifficulty1Input = new ArrayList<>(Arrays.asList("X", "O", "2", "1", "2", "3"));
+        MockUi ui = new MockUi(aiDifficulty1Input);
+        Config config = new Config(ui);
+        String difficulty = config.getDifficulty();
+
+        assertEquals("Medium", difficulty);
```
Comment (GeorgeSmith-Sweeper) [inline src/test/java/com/EighthLight/app/TestConfig.java:180]: Simple assertion to check if the difficulty field gets set to the correct value

File: src/main/java/com/EighthLight/app/Constants.java [hunk 3]
```diff
@@ -10,14 +10,15 @@
     public static final String INVALID_SPOT_MSG = "Invalid spot, pick again!";
     public static final String INCORRECT_INPUT = "incorrect input";
     public static final String CORRECT_INPUT = "correct input";
-    public static final String GAME_MODE_PROMPT = "Please choose a game mode. (1)Human vs Human, (2)Human vs Ai";
+    public static final String GAME_MODE_PROMPT = "Please choose a game mode. 1) Human vs Human\n" +
```
Comment (GeorgeSmith-Sweeper) [inline src/main/java/com/EighthLight/app/Constants.java:13]: Put prompts with more then one option on separate lines

**Review by JeromeGoodrich — COMMENTED** [review]
🚀

**Pull request merged**